ArduPlane/Parameters.h

```diff
@@ -93,14 +93,14 @@ class Parameters {
         k_param_scheduler,
         k_param_relay,
         k_param_takeoff_throttle_delay,
-        k_param_skip_gyro_cal, // unused
+        k_param_takeoff_rudder_mix,
```

Comment:
@bugobliterator If an index was used and then removed it can contain a saved value, independently of it being transitioned to a new parameter or not. If you re-use that index when loading the saved values your new parameter will have a bad value.